libcontainer/cgroups/utils.go

```diff
@@ -479,6 +479,26 @@ func RemovePaths(paths map[string]string) (err error) {
 	return fmt.Errorf("Failed to remove paths: %v", paths)
 }
 
+// RemovePathUnified aims to remove cgroup path recursively
+// Because there may be subcgroups in it.
+func RemovePathUnified(path string) error {
+	infos, err := ioutil.ReadDir(path)
+	if err == nil {
```

Comment:
1 nit:
```go
if err != nil {
   if os.IsNotExist(err) {
      err = nil
   }
   return err
}
```

so
1. the rest of function will be less indented
2. we won't return ENOENT.